<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">boder:none;</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="process">
     <property name="styleSheet">
      <string notr="true">border:0px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QTableWidget" name="processTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>800</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget::item
{
	border-right:1px solid rgb(252, 175, 62);
}

QHeaderView
{
	min-height: 50px;
	border:2.2px solid rgb(136, 138, 133);
	border-left:none;
	border-right:none;
	border-top:none;
}

QHeaderView::section {	
	color: black;
	padding: 1px 1px 1px 1px;
	background-color: rgba(255,255,255,100%);
	border:2px solid rgba(136, 138, 133, 255);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(136, 138, 133, 255), stop:1 rgb(243, 243, 243,255));
	border-left:none;
	border-bottom:none;
	border-top:none;
}

QScrollBar:vertical{
    width:25px;
    border:none;
    margin:0px,0px,0px,0px;
    padding-top:25px;
    padding-bottom:25px;
	border-color: rgba(0, 0, 0,10%);
}

QScrollBar::handle:vertical
{
    width:25px;
	border:none;
    background:rgba(0,0,0,25%);
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    background:rgba(0,0,0,40%);
}

QScrollBar::add-line:vertical
{
	width:25px;
	height:25px;
	background:rgba(0,0,0,10%);
	border-image: url(:/images/arrow-down-1.png);
	subcontrol-position: center bottom;
	left:-5px;
}
QScrollBar::sub-line:vertical
{
	width:25px;
	height:25px;
	background: rgba(0,0,0,10%);
	border-image: url(:/images/arrow-up-1.png);
	subcontrol-position: center top;
	left:-5px;
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(0,0,0,10%);
}

QHeaderView::up-arrow { 
subcontrol-position: center top;
width: 15px;
height: 15px;
top: 5px;
}
QHeaderView::down-arrow { 
subcontrol-position: center top;
width: 15px;
height: 15px;
top: 5px;
}
</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignBottom</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>名称</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignBottom</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignBottom</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CPU</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignBottom</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>内存</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignBottom</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>优先级</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignBottom</set>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="resource">
     <property name="styleSheet">
      <string notr="true">border:0px;
background-color: rgb(243, 243, 243);</string>
     </property>
     <attribute name="title">
      <string>资源使用</string>
     </attribute>
     <widget class="QTabWidget" name="resources">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>776</width>
        <height>540</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="CPU">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <widget class="QWidget" name="cpuUse" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>700</width>
          <height>250</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(114, 159, 207);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>% 利用率</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>40</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>100%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="modelname">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>400</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="cpumenu">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>340</y>
          <width>100</width>
          <height>150</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>YaHei Consolas Hybrid</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QLabel" name="cpuinfo">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>340</y>
          <width>160</width>
          <height>150</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>YaHei Consolas Hybrid</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QLabel" name="rateTitle">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>YaHei Consolas Hybrid</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>利用率</string>
        </property>
       </widget>
       <widget class="QLabel" name="speedTitle">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>340</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>YaHei Consolas Hybrid</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>速度</string>
        </property>
       </widget>
       <widget class="QLabel" name="useValue">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>360</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Light</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="speedValue">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>360</y>
          <width>100</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Light</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="processTitle">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>410</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>YaHei Consolas Hybrid</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>进程</string>
        </property>
       </widget>
       <widget class="QLabel" name="processValue">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>430</y>
          <width>100</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Light</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="RAM">
       <attribute name="title">
        <string>内存</string>
       </attribute>
       <widget class="QWidget" name="ramUse" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>700</width>
          <height>250</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(114, 159, 207);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>5</y>
          <width>80</width>
          <height>35</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>内存</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>内存使用量</string>
        </property>
       </widget>
       <widget class="QLabel" name="totalsize">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>40</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="raminfo">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>340</y>
          <width>160</width>
          <height>150</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>YaHei Consolas Hybrid</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QLabel" name="rammenu">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>340</y>
          <width>100</width>
          <height>150</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>YaHei Consolas Hybrid</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="system">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QLabel" name="osmenu">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>120</width>
        <height>180</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QLabel" name="ubuntu">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
image: url(:/images/ubuntu.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="os_version">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="osinfo">
      <property name="geometry">
       <rect>
        <x>245</x>
        <y>240</y>
        <width>420</width>
        <height>180</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="about">
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QLabel" name="aboutinfo">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>400</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选项</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
